src/v2/controllers/FormController.js

```diff
@@ -143,7 +143,7 @@ export default Controller.extend({
       idx.proceed(actionPath, {})
         .then(this.handleIdxSuccess.bind(this))
         .catch(error => {
-          this.showFormErrors(this.formView.model, error);
+          this.showFormErrors(this.formView.model, error, this.formView.form);
```

Comment:
we can save an extra argument if we just pass `this.formView` as first argument.. and extract `model` in the function

Reply:
`handleSaveForm` line 166 receives the `model` as an argument, so I don't want to change that. It is probably ok, but I don't want to introduce any risk in case anyone calling that method (which is called when we trigger 'saveForm` event on the appState) needs to pass a different model

Reply:
`showFormErrors` only used in `FormController`..  your choice though. 👍